ui/app/AppLayouts/Chat/CommunityComponents/CommunityProfilePopupMembersList.qml

```diff
@@ -18,6 +19,7 @@ Item {
         //% "Invite People"
         label: qsTrId("invite-people")
         width: parent.width
+        type: globalSettings.theme === Universal.Dark ? "secondary" : "primary"
```

Comment:
This is a rather (horrible) hack to get the desired colors in Darkmode (and even that is only half the truth).

This component uses `StatusRoundButton` under the hood to render an icon as part of the menu item. Obviously that's already semantically incorrect and needs to be refactored, but that is a bigger effort. 

The `StatusRoundButton` distinguishes between `primary` and `secondary` types. The types have an effect on the look and feel of the component. It just so happens that, per design, in light mode the icon looks like a `primary` round button, however in dark mode, it looks like a `secondary` round button.

So for now I'm using this hack to get the UI that's needed, but in to fix this properly we need to redo this component and introduce a `StatusRoundIcon` component (or something similar) that isn't restricted to `primary` and `secondary`.

Also, the colors aren't 100% correct,  because the design actually expected a color with transparency, so that when the menu item is hovered, the icon background changes along with it (something that's not implemented in `StatusRoundButton` either). 

Again, this needs more refactoring, so consider this a quick fix.

I've jotted down some of those thoughts in #2295